sdk-docs/testing-iris-apps/testing-iris-apps.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Testing Iris App
+category: 61fcd8e1a448f5004215317c
+parentDocSlug: testing-iris-apps
+---
+
+> 🚧 Beta
+> 
+> This is a beta version and subject to change without notice. Pricing, terms, conditions and availability may change in the final version.
+
+We highly recommend reading through this section before you start developing your Iris App. It will give you a good understanding of the testing effort for Iris App SDK and how to use it.
+
+## Testing your Iris App
+
+### 1. Learn about standard testing in react using the React Testing Library:
```

Comment:
Maybe give me pointers on which 2-3 concepts in the external guide are most important to succeed testing the Iris App. If I just want to skim it and quickly get an overview, it can be helpful to know this.